src/common/options/global.yaml.in

```diff
@@ -5021,14 +5021,15 @@ options:
   default: 0
   with_legacy: true
 - name: bluestore_debug_legacy_omap
-  type: bool
+  type: str
   level: dev
-  desc: Allows mkfs to create OSD in legacy OMAP naming mode (neither per-pool nor per-pg).
+  desc: Create omap in legacy mode
+  long_desc: Allows mkfs to create OSD in legacy OMAP naming mode (neither per-pool nor per-pg).
     This is intended primarily for developers' purposes. The resulting OSD might/would
     be transformed to the currrently default 'per-pg' format when BlueStore's quick-fix or
-    repair are applied.
-  default: false
-  with_legacy: true
+    repair are applied. Acceptable values 'bulk', 'per-pool'.
```

Comment:
There is an "enum_values" clause which adds permitted values. Not sure whether it works with empty string though...

Reply:
How about dropping "legacy" and make "bluestore_omap" with values "bulk" "per-pool" "per-pg" ?

Reply:
sounds good

Reply:
may be rename to bluestore-omap-naming?